pom.xml

```diff
@@ -14,28 +14,32 @@
   <url>https://github.com/jenkinsci/${project.artifactId}-plugin</url>
 
   <scm>
-    <connection>scm:git:git://github.com/jenkinsci/${project.artifactId}-plugin.git</connection>
-    <developerConnection>scm:git:git@github.com:jenkinsci/${project.artifactId}-plugin.git</developerConnection>
+    <connection>scm:git:https://github.com/${gitHubRepo}.git</connection>
+    <developerConnection>scm:git:git@github.com:${gitHubRepo}.git</developerConnection>
     <url>https://github.com/jenkinsci/${project.artifactId}-plugin</url>
     <tag>${scmTag}</tag>
   </scm>
 
   <properties>
     <revision>1.14</revision>
     <changelist>-SNAPSHOT</changelist>
-    <jenkins.version>2.249.3</jenkins.version>
-    <java.level>8</java.level>
+    <jenkins.version>2.289.3</jenkins.version>
   </properties>
 
   <dependencyManagement>
     <dependencies>
       <dependency>
         <groupId>io.jenkins.tools.bom</groupId>
-        <artifactId>bom-2.249.x</artifactId>
-        <version>961.vf0c9f6f59827</version>
+        <artifactId>bom-2.289.x</artifactId>
+        <version>1289.v5c4b_1c43511b_</version>
         <scope>import</scope>
         <type>pom</type>
       </dependency>
+      <dependency>
+        <groupId>org.apache.commons</groupId>
+        <artifactId>commons-lang3</artifactId>
```

Comment:
Mm, reminds me that this really needs to be made into a library wrapper plugin.

Reply:
https://github.com/jenkinsci/commons-lang3-api-plugin ?